src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -324,6 +324,14 @@ std::optional<syncer::RaySyncMessage> LocalResourceManager::CreateSyncMessage(
 
   resources_data.set_is_draining(IsLocalNodeDraining());
 
+  std::stringstream node_activity;
```

Comment:
probably also need to think about how this would show up with the leased workers in the other PR.